### PR TITLE
Fix a bad link and document DiffDelta type

### PR DESCRIPTION
**Pull request by kshyatt**
This should link to `delta` not to `file`.

### CODE REVIEW

**Review by ararslan — APPROVED** [review]
Nice, good catch.

File: base/libgit2/types.jl [hunk 1]
```diff
@@ -298,7 +298,18 @@ end
     LibGit2.DiffDelta
 
 Description of changes to one entry.
-Matches the [`git_diff_file`](https://libgit2.github.com/libgit2/#HEAD/type/git_diff_file) struct.
+Matches the [`git_diff_delta`](https://libgit2.github.com/libgit2/#HEAD/type/git_diff_delta) struct.
+
+The fields represent:
+  * `status`: One of `Consts.DELTA_STATUS`, indicating whether the file has been added/modified/deleted.
+  * `flags`: Flags for the delta and the objects on each side. Determines whether to treat the file(s)
+     as binary/text, whether they exist on this side of the diff, and whether the object ids are known
```
Comment (tkelman) [inline base/libgit2/types.jl]: which side is "this side"?

File: base/libgit2/types.jl [hunk 2]
```diff
@@ -298,7 +298,18 @@ end
...
+     to be correct.
+  * `similarity`: Used to indicated if a file has been renamed or copied.
```
Comment (tkelman) [inline base/libgit2/types.jl]: to indicate

**Review by Sacha0 — APPROVED** [review]
lgtm modulo `tkelman`'s comments! :)

**Comment by kshyatt** [comment]
@tkelman good to go?

**Review by ararslan — APPROVED** [review]
Double approval

**Comment by Sacha0** [comment]
The updated commit seems to address `tkelman`'s comments, so absent objections, requests for time, or someone beating me to it, I plan to merge this tomorrow morning. Best!

**Pull request merged**